source/Weather.cpp

```diff
@@ -142,6 +142,14 @@ void Weather::CalculateStrength()
 
 
 
+// Get information on how this hazard impacted a ship.
+Weather::ImpactInfo Weather::GetInfo() const
+{
+	return ImpactInfo(*hazard, origin, DamageMultiplier());
```

Comment:
Does anyone else need to access `DamageMultiplier` now? If not, make it private.